src/Command/Docker/Restart.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace eZ\Launchpad\Command\Docker;
+
+use eZ\Launchpad\Core\DockerCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class Restart.
+ */
+final class Restart extends DockerCommand
+{
+    protected function configure(): void
+    {
+        parent::configure();
+        $this->setName('docker:start')->setDescription('Restart all the services (or just one).');
```

Comment:
`docker:start`? I think `docker:restart` would be better!

Reply:
It is done. Thank you, I appreciate for the code review 🙏